.github/workflows/react_tests.yml

```diff
@@ -1,45 +1,23 @@
-name: React Tests (non-blocking)
```

Comment:
This, albeit the name, used to be a required check in GH.
After the rename, it's not matching anymore, so you get the "waiting" job.

I don't think this *needs* to be a "required check", but if you want it to be, it needs to run on every PR, not limited to `paths` like below.

Reply:
Ahhh, I thought it was my rename of `build` to `react-tests`. That explains why changing it back to `build` didn't work.

@parthaa I would like to call this "React Tests" (without the "(non-blocking)"). Can you please make "React Tests" the required check and remove "React Tests (non-blocking)" ?

Reply:
@jeremylenz you probably need to merge this first before you can add it as a required check.